packages/samples/headless-react/src/App.tsx

```diff
@@ -58,6 +62,7 @@ import {
 } from '@coveo/headless';
 import {bindSearchParametersToURI} from './components/search-parameter-manager/search-parameter-manager';
 import {setContext} from './components/context/context';
+import filesize from 'filesize';
```

Comment:
I thought it would be somewhat nice to show a clean way to format numbers such as file sizes in those samples.